packages/account/src/Sections/Security/TwoFactorAuthentication/__test__/two-factor-authentication.spec.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { cleanup, render, screen, waitFor } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { WS } from '@deriv/shared';
+import TwoFactorAuthentication from '../two-factor-authentication';
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    WS: {
+        authorized: {
+            accountSecurity: jest.fn().mockResolvedValue({}),
+        },
+    },
+}));
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+
+    return {
+        ...original_module,
+        Loading: jest.fn(() => 'mockedLoading'),
+    };
+});
+
+jest.mock('qrcode.react', () => jest.fn(() => <div>QRCode</div>));
+
+describe('<TwoFactorAuthentication/>', () => {
+    afterEach(() => {
+        cleanup();
+        jest.clearAllMocks();
+    });
+
+    const store = mockStore({
+        client: {
```

Comment:
here we also can skip some of default values as you  added 'email_address' and 'getTwoFAStatus' fields to the mockStore

Reply:
We still need to specify getTwoStatus but we can remove `email_address` and `is_switching`.